Extension/src/LanguageServer/configurations.ts

```diff
@@ -965,6 +966,29 @@ export class CppProperties {
                     this.lastCustomBrowseConfiguration.Value = undefined;
                 }
             }
+
+            /*
+             * Ensure all paths are absolute
+             */
+            if (configuration.macFrameworkPath) {
+                configuration.macFrameworkPath = configuration.macFrameworkPath.map((path: string) => this.resolvePath(path, this.isWin32));
+            }
+
+            if (configuration.dotConfig) {
+                configuration.dotConfig = this.resolvePath(configuration.dotConfig, this.isWin32);
+            }
+
+            if (configuration.compileCommands) {
+                configuration.compileCommands = this.resolvePath(configuration.compileCommands, this.isWin32);
+            }
+
+            if (configuration.forcedInclude) {
+                configuration.forcedInclude = configuration.forcedInclude.map((path: string) => this.resolvePath(path, this.isWin32));
+            }
+
+            if (configuration.includePath) {
+                configuration.includePath = configuration.includePath.map((path: string) => this.resolvePath(path, this.isWin32));
```

Comment:
@Colengms @Bernardin-MS  This broke recursive includes because `${workspaceFolder}/**` has the `**` removed. I'm looking into a fix.

Reply:
I have a PR with a fix at https://github.com/microsoft/vscode-cpptools/pull/10337